app/v2/controllers/actions/AcceptHeaderAction.scala

```diff
@@ -31,7 +31,8 @@ import scala.concurrent.Future
 
 trait AcceptHeaderAction[R[_] <: Request[_]] extends ActionRefiner[R, R]
 
-class AcceptHeaderActionImpl[R[_] <: Request[_]] @Inject() (implicit val executionContext: ExecutionContext) extends AcceptHeaderAction[R] {
+class AcceptHeaderActionImpl[R[_] <: Request[_]] @Inject() (isOnlyJson: Boolean)(implicit val executionContext: ExecutionContext)
```

Comment:
It might be better to rename the parameter `acceptOnlyJson`, because that's more indicative of what we're doing.

Reply:
Sure updated